src/System.Private.CoreLib/src/Microsoft/Win32/SafeHandles/SafeRegistryHandle.Windows.cs

```diff
@@ -1,19 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using System.Runtime.InteropServices;
-
-namespace Microsoft.Win32.SafeHandles
-{
-#if REGISTRY_ASSEMBLY
```

Comment:
These files are also in corefx. https://github.com/dotnet/corefx/blob/master/src/Microsoft.Win32.Registry/src/Microsoft/Win32/SafeHandles/SafeRegistryHandle.cs

I think you should unify them with the CoreFX copy too - it requires these ifdefs to stay.

Reply:
I will add the if defs, I will delete the files from corefx after the mirroring PR